src/components/CurratedContentCard/index.stories.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+import { CurratedContentCard } from '.'
+
+export default {
+  title: 'Components/Data Display/CurratedContentCard',
+}
```

Comment:
Could you create a variable for this object, and export that variable? That fixes a warning 😄

Reply:
Also, this should be in surfaces category